cortex/secondary/call_degree.py

```diff
@@ -1,30 +1,57 @@
 """ Module for call_degree from raw feature calls """
 import numpy as np
-
+import logging
+import pdb
 from ..feature_types import secondary_feature
-from ..raw.calls import calls
+from ..raw.telephony import telephony
 
 MS_IN_A_DAY = 86400000
 @secondary_feature(
     name='cortex.feature.call_degree',
-    dependencies=[calls]
+    dependencies=[telephony]
 )
-def call_degree(**kwargs):
-    """Returns the number of unique phone numbers a participant called
 
+def call_degree(call_type,**kwargs):
```

Comment:
I think it may be a good idea to set the default value of `call_type` to 'incoming' to more closely match the original behavior of this feature - or perhaps to 'all'.

It would also be a good idea to add an `isinstance` check to see if someone is trying to pass a boolean to `incoming` and use that to preserve the prior behavior (e.g. if they pass `True` return incoming calls, else return outgoing). This maintains more backwards compatibility. Moreover it would be useful to check that they passed a valid input, and return None (log a warning) if not.

Finally, can you change `call_type` to be named`call_direction`? This will keep it consistent across features.